src/raven_logger_backend.erl

```diff
@@ -56,13 +56,25 @@ is_httpc_log(#{meta := Meta} = _LogEvent) ->
 				 Report =:= fun ssl_logger:format/1
 	end.
 
+is_domain(Domain, #{ domain := Domains }) ->
```

Comment:
code-style nit: called functions should appear after first calling (ie, this should be below get_msg/1)